js/showdown-extensions.js

```diff
@@ -21,18 +21,18 @@ function wbSlugify(obj) {
 
 // This extension is a template-like mechanism, allowing
 // to replace variables by a static content.
-// For example, the markdown string "{{ date.year }}" is replaced by "2016"
-showdown.extension("wbVariables", function() {
+// For example, the markdown string '{{ date.year }}' is replaced by '2016'
+showdown.extension('wbVariables', function() {
   // static variables to maintain
   // TODO: could be computed
   var vars = {
     webots : {
       version : {
-        major : "R2018a",
+        major : 'R2018a',
```

Comment:
Note that this are updated through a script and the script has to be updated as well otherwise it won't work: `src/bin/new_version.sh`